Dockerfile

```diff
@@ -18,27 +18,32 @@ RUN echo msodbcsql17 msodbcsql/ACCEPT_EULA boolean true | debconf-set-selections
 
 # This section to installs additional packages.
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-    && apt-get -y install --no-install-recommends \
-    build-essential \
-    unixodbc-dev \
-    msodbcsql17 \
-    # odbc-mariadb/buster-backports \
-    sqlite3 \
-    libsqliteodbc
+	&& apt-get -y install --no-install-recommends \
+	build-essential \
+	unixodbc-dev \
+	msodbcsql17 \
+	# odbc-mariadb/buster-backports \
+	sqlite3 \
+	libsqliteodbc
 
 # Fix SQLite driver paths
 RUN sed --in-place 's/libsqlite3odbc.so/\/usr\/lib\/x86_64-linux-gnu\/odbc\/libsqlite3odbc.so/' /etc/odbcinst.ini
 RUN sed --in-place 's/libsqliteodbc.so/\/usr\/lib\/x86_64-linux-gnu\/odbc\/libsqliteodbc.so/' /etc/odbcinst.ini
 
 # Install MariaDB driver from tar bundle
-COPY mariadb-connector-odbc-3.1.11-debian-buster-amd64.tar.gz .
+COPY docker/dev/mariadb-connector-odbc-3.1.11-debian-buster-amd64.tar.gz .
+COPY docker/dev/mariadb_odbc_template.ini .
 RUN tar -xf mariadb-connector-odbc-3.1.11-debian-buster-amd64.tar.gz
 RUN cp mariadb-connector-odbc-3.1.11-debian-buster-amd64/lib/mariadb/libmaodbc.so /usr/lib/x86_64-linux-gnu/odbc/libmaodbc.so
-# RUN sed --in-place 's/libmaodbc.so/\/usr\/lib\/x86_64-linux-gnu\/odbc\/libmaodbc.so/' /etc/odbcinst.ini
-
-# Setup cargo and install cargo packages required for tests
-USER vscode
+RUN odbcinst -i -d -f mariadb_odbc_template.ini
 
 # There is also a rust devcontainer, yet this way we get a toolchain
 # which is updatable with rustup.
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --default-toolchain stable -y
+
+# Setup workspace
+WORKDIR /workspace
+COPY . .

```

Comment:
Will this copy the entire repository into the container? At least for local development I think it's nicer to mount it.

How do you feel about having a seperate container for local development and ci?

E.g.
```
repo
|
|-docker-compose-local.yml
|-docker-compose-ci.yml
|-docker
  |- mariadb_odbc_template.ini
  |- Dockerfile-local
  |- Dockerfile-ci
```

Reply:
I'm not gonna add an additional docker-compose file because that just makes the commands longer. But I'm gonna restore the original "lite" image for development. The Dockerfile in the root will be the CI one (because it can't copy anything in the container that is outside of the CWD), the other one will be the development one